File: src/main/java/ink/rubi/codegenerator/po/DbQueryItem.java
package ink.rubi.codegenerator.po;

import com.baomidou.mybatisplus.generator.config.IDbQuery;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author : Rubi
 * @version : 2019-05-18 01:53 上午
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class DbQueryItem implements Item<IDbQuery> {
    private IDbQuery dbQuery;
    private String description;

    @Override
    public IDbQuery getValue() {
        return this.dbQuery;
    }

    @Override
    public String getDesc() {
        return this.description;
    }
}

File: src/main/java/ink/rubi/coffee/po/DbQueryItem.kt
package ink.rubi.coffee.po

import com.baomidou.mybatisplus.generator.config.IDbQuery

/**
 * @author : Rubi
 * @version : 2019-05-18 01:53 上午
 */
data class DbQueryItem(override var value: IDbQuery? = null, override var desc: String? = null) : Item<IDbQuery>
